draft-irtf-cfrg-frost.md

```diff
@@ -802,9 +802,9 @@ signature using the following procedure.
     group_commitment = compute_group_commitment(commitment_list, binding_factor_list)
 
     # Compute aggregated signature
-    z = 0
+    z = Scalar(0)
     for z_i in sig_shares:
-      z = z + z_i
+      z += z_i
```

Comment:
@chris-wood we don't define the Scalar += operation in this document, do we need to?

Reply:
Oh, you're right ! I did that change to harmonize the usage of this operator throughout function definitions, but did not consider the nature of the receivers.

Reply:
Good catch. I reverted the change.